app/views/layouts/darkswarm.html.haml

```diff
@@ -50,4 +33,23 @@
 
           #footer
     %loading
+
+    %script{:src => "https://js.stripe.com/v3/", :type => "text/javascript"}
+    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry#{ ENV['GOOGLE_MAPS_API_KEY'] ? '&key=' + ENV['GOOGLE_MAPS_API_KEY'] : ''} "}
+    = javascript_include_tag "darkswarm/all"
```

Comment:
I am not 100% sure this is possible. Are the pages rendering normally? Maybe we need to test with a very slow internet connection.

Reply:
I did some very brief testing and looking around, and it all seemed to be working.